db/migrations/0014_add_provider_rsr_scores.sql

```diff
@@ -0,0 +1,25 @@
+-- Migration number: 0014 	 2025-01-27T00:00:00.000Z
+CREATE TABLE IF NOT EXISTS provider_rsr_scores (
+  provider_address TEXT NOT NULL,
+  score REAL NOT NULL,
```

Comment:
```suggestion
```

Let's remove this score for now as we can calculate that on the fly based o `total_requests` and `successful_requests`.